lib/gosquared/index.js

```diff
@@ -90,21 +91,62 @@ GoSquared.prototype.page = function(page){
 
 GoSquared.prototype.identify = function(identify){
   var traits = identify.traits({
-    userId: 'user_id',
-    created: 'created_at'
+    createdAt: 'created_at',
+    firstName: 'first_name',
+    lastName: 'last_name',
+    title: 'company_position',
+    industry: 'company_industry'
+  });
+
+  // https://beta.gosquared.com/docs/tracking/api/#properties
+  var special = [
+    'id',
+    'email',
+    'name',
+    'first_name',
+    'last_name',
+    'username',
+    'description',
+    'avatar',
+    'phone',
+    'created_at',
+    'company_name',
+    'company_size',
+    'company_position',
+    'company_industry'
+  ];
```

Comment:
Nitpick: Could we rename this to `specialKeys`? Just a little easier to understand :)